packages/protocol/src/abstracts/BaseVault.sol

```diff
@@ -34,6 +34,10 @@ abstract contract BaseVault is ERC20, SystemAccessControl, PausableVault, VaultP
   error BaseVault__redeem_invalidInput();
   error BaseVault__setter_invalidInput();
   error BaseVault__checkRebalanceFee_excessFee();
+  error BaseVault__depositSlippageProtection();
+  error BaseVault__mintSlippageProtection();
+  error BaseVault__withdrawSlippageProtection();
+  error BaseVault__redeemSlippageProtection();
```

Comment:
Rename according to the convention. I'd suggest for the error description to use "slippageTooHigh", for ex. `BaseVault__deposit_slippageTooHigh()`

Reply:
fixed as suggested.